src/main/java/com/microsoft/sqlserver/jdbc/Geography.java

```diff
@@ -403,10 +417,11 @@ protected void parseWkb() {
     }
 
     private void readPoints() {
```

Comment:
Looks like a few methods are duplicated in both Geometry/Geography classes and they should be moved to SQLServerSpatialDatatype.

1.  parseWkb(), exact duplicate. 
2.  readPoints(), serializeToWkb()  - The only difference is you have to read yValues before xValues. Can you add some kind of a check and move these methods to SQLServerSpatialDatatype too?
3. There is a lot of duplication in other methods too, please take a look and move to SQLServerSpatialDatatype when possible.